### PR TITLE
Fix clean initialization of FormatPlugin

### PR DESCRIPTION
**Pull request by PeterSuna**
Signed-off-by: Peter Suna <peter.suna@pantheon.tech>

### CODE REVIEW

File: src/main/java/io/lighty/yang/validator/formats/JsonTree.java [hunk 1]
```diff
@@ -90,12 +90,13 @@
     private static final String SLASH = "/";
     private static final String COLON = ":";
 
-    private final Map<String, String> prefixMap = new HashMap<>();
+    private Map<String, String> prefixMap = new HashMap<>();
```
Comment (vainyksi) [inline src/main/java/io/lighty/yang/validator/formats/JsonTree.java:93]: it looks like it can be changed to local variable

**Review by vainyksi — CHANGES_REQUESTED** [review]

**Review by vainyksi — APPROVED** [review]

File: src/main/java/io/lighty/yang/validator/formats/JsonTree.java (hunk 1 above)
Comment (vainyksi) [reply src/main/java/io/lighty/yang/validator/formats/JsonTree.java:93]: hm... values are put int map only, no no getting vealues from map, so it looks like the variable is useless here

Comment (vainyksi) [reply src/main/java/io/lighty/yang/validator/formats/JsonTree.java:93]: hm... values are put int map only, no no getting vealues from map, so it looks like the variable is useless here

Comment (PeterSuna) [reply src/main/java/io/lighty/yang/validator/formats/JsonTree.java]: done

**Review by vainyksi — APPROVED** [review]

**Pull request merged**